developer-setup.md

```diff
@@ -27,9 +27,11 @@
   ```
 * Make sure that you have cloned cstor code in the system. Refer [cstor setup](https://github.com/openebs/cstor/blob/master/developer-setup.md)
```

Comment:
cstor setup link should be `https://github.com/openebs/cstor/blob/develop/developer-setup.md`

Reply:
Updated